src/editors/playgroundController.ts

```diff
@@ -316,13 +317,38 @@ export default class PlaygroundController {
     return this._createPlaygroundFileWithContent(content);
   }
 
+  async createPlaygroundFromTreeView(
+    treeItem: DatabaseTreeItem | CollectionTreeItem
+  ): Promise<boolean> {
+    let content = '';
+    if (treeItem instanceof DatabaseTreeItem) {
+      content = playgroundInsertDocumentTemplate.replace(
+        'CURRENT_DATABASE',
+        treeItem.databaseName
+      );
+      this._telemetryService.trackPlaygroundCreated('fromDatabaseTreeItem');
+    } else if (treeItem instanceof CollectionTreeItem) {
+      content = playgroundInsertDocumentTemplate
+        .replace('CURRENT_DATABASE', treeItem.databaseName)
+        .replace('CURRENT_COLLECTION', treeItem.collectionName);
```

Comment:
Looks like in the `src/templates/playgroundCreateCollectionTemplate.ts` we keep using the `NEW_COLLECTION_NAME` name, not `CURRENT_COLLECTION`.